src/codecoverage/bot/tests/test_artifacts.py

```diff
@@ -27,6 +27,14 @@ def FAKE_ARTIFACTS_DIR(tmpdir):
     return tmpdir.strpath
 
 
+def test_generate_path(FAKE_ARTIFACTS_DIR):
+    a = ArtifactsHandler([], parent_dir=FAKE_ARTIFACTS_DIR)
+    artifact = {'name': 'code-coverage-jsvm.info'}
```

Comment:
Nit: assign a value to `artifact` then test generate_path, then overwrite `artifact` and test it again. This way we don't need the `artifact2` variable. The other option is to rename `artifact` to `artifact_jsvm` and `artifact2` to `artifact_grcov`, which would make the variable names more meaningful.